particle-iot/device-os
# rtl872x: revert the flash speed to 80MHz

### Problem

This is a mistake I made in [PR2470](https://github.com/particle-iot/device-os/pull/2470)

The older datasheet indicates support for QSPI speeds of up to 60MHz, whereas the newer one indicates support for speeds of up to 80MHz.

<img width="1455" alt="Screenshot 2023-08-15 at 14 59 02" src="https://github.com/particle-iot/device-os/assets/7424522/ac668de4-613c-4bff-acf7-5c3981a859ef">

Realtek also tested MX25R32 with 80MHz speed:
![Screenshot 2023-08-15 at 15 08 44](https://github.com/particle-iot/device-os/assets/7424522/05951e30-fa86-4bea-8e69-b13381e00496)


[UM0404 Ameba Flash AVL-EN_20210621(45695).pdf](https://github.com/particle-iot/device-os/files/12341967/UM0404.Ameba.Flash.AVL-EN_20210621.45695.pdf)
[MX25R6435F v0.03.pdf](https://github.com/particle-iot/device-os/files/12341972/MX25R6435F.v0.03.pdf)
[MX25R6435F v1.6.pdf](https://github.com/particle-iot/device-os/files/12341974/MX25R6435F.v1.6.pdf)


### Solution

Revert the flash speed from 57MHz to 80MHz.

**NOTE: This PR requires updating the prebootloader-part1!**

### Steps to Test

1. Update the prebootloader to observe the flash clock with an oscilloscope
1. Update prebootloader with the change in this PR, use bootloader/system-part without this changeset
1. Update bootloader
1. Update system-part


### References

N/A

---

### Completeness
